Assets/MixedRealityToolkit/MixedReality.Toolkit.targets

```diff
@@ -1,14 +1,32 @@
 <Project>
-
-  <PropertyGroup>
+  <!-- Check if MSBuildForUnity is being used to resolve, by checking if it's version is present or not -->
```

Comment:
```suggestion
  <!-- Check if MSBuildForUnity is being used to resolve, by checking if its version is present or not -->
```

Reply:
Yes it should be, I'll retarget to 2.3 stabilization.